apple/RNGestureHandlerModule.mm

```diff
@@ -51,6 +51,8 @@ @implementation RNGestureHandlerModule {
 
   // Oparations called after views have been updated.
   NSMutableArray<GestureHandlerOperation> *_operations;
+
+  jsi::Runtime *_runtime;
```

Comment:
Since we're accessing both RN and UI runtimes, I'd suggest using `rnRuntime` and `uiRuntime` instead.
```suggestion
  jsi::Runtime *_rnRuntime;
```